Accepted-RFCs/RFC-014-product-vision-templates-1.0.md

```diff
@@ -0,0 +1,196 @@
+
+This proposal has us releasing "templates 1.0" in late 2023, with a 0.9 shortly before WTD Portland 2023.
+
+### Side note about the word "product"
+
+Some people don't like the word "product," but broadly speaking, we're word nerds here. We're aiming to *produce* a set of usable templates, and our templates are set up as milestone-informed packages. That's a product—not commercially, but essentially. So we'll use that term to separate user-facing deliverables like templates (and later doc tools) from non-products like the blog, news, etc.
+
+## Motivation
+
+As an organization, we've been talking about having template releases for years. We've been actively recruiting people to help participate in the process, but we've lacked a critical element: an overarching plan for what's a "1.0" release of our templates.
+
+On a logisitcs level, it feels hard to pitch our project to people, as we've spent multiple WTD Portlands talking about goals and aspirations, but not providing someone concrete that proves The Good Docs Project isn't just vaporware. As an organization, we could use the credibility of achieving a concrete 1.0 launch.
+
+And if we're aiming to support devs who are overwhelmed or scared to make docs, they could use that 1.0 launch too.
+
+Another very important benefit of having a product plan/roadmap is that it'll help us better identify what template work needs to be done (thus filling in our backlog and issue list). We have a perpetual problem of new community members who join our project and are excited to create templates, but very few of them have their own independent ideas for what templates they could work on. Many templateers would prefer for us to say what work needs to be done and what our priorities are. Much of the time, we just make this up and talk it out on an ad-hoc basis, but it would be great to have a clear plan that clearly indicates what work needs to be done.
+
+So basically, a product plan/roadmap will supercharge our productivity and improve our onboarding process for new templateers, reducing the burden on template leads to invent work for new community members to do on an ad-hoc basis.
+
+## Proposal
+
+Simply put: It's time to decide as a group what *should* be in "Templates 1.0." Rather than working solely bottom-up, with people deciding what templates inspire them or working on other projects, we should add *some* top-down direction to shape the work.
+
+This includes (as post-RFC decisions):
+ * Defining our definition of "done & confidently releasable" for individual templates
+ * Deciding what meets our quality standards and acceptance criteria
+
+### What makes up the template product 1.0
+
+- Whatever templates we deem as "core" (which is a follow-up conversation)
```

Comment:
Food for thought for the follow-up conversation: 
@barbaricyawps  Did a great job with sketching out "template packs". I think they are a wonderful grouping of templates. We could use that as a reference as to which of the pack(s) to include in 1.0.